src/app/todos/todos.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, OnInit } from '@angular/core';
+import { Todo } from '../todo';
+import { TODOS } from '../mock-todos';
+import { TodoService } from '../todo.service';
+
+@Component({
+  selector: 'app-todos',
+  templateUrl: './todos.component.html',
+  styleUrls: ['./todos.component.css']
+})
+export class TodosComponent implements OnInit {
+
+  public todos: Todo[];
+  public selectedTodo: Todo;
+
+  constructor(private todoService: TodoService) { }
+
+  public getTodos(): void {
+/*    this.todoService.getTodos()
+      .subscribe(todos => this.todos = todos);
+    console.log(this.todos);*/
+    this.todos = TODOS;
```

Comment:
Der TodoService erscheint mir als ziemlich überflüssig, wenn du hier eh die statischen To-Dos aus `mock-todos` lädst.

Reply:
Eigentlich war ich schon weiter und hatte mich von den mock-todos gelöst und mit HTTP Request gearbeitet. Nur gab es ein Cors Policy Problem und wir haben das nochmal vorläufig auf mock-todos umgestellt. 
https://github.com/martinfruehauf/todo-angular/pull/1#discussion_r434581910

Reply:
Ach so.